winml/lib/Api.Ort/OnnxruntimeModel.cpp

```diff
@@ -130,6 +130,11 @@ STDMETHODIMP ModelInfo::GetName(const char** out, size_t* len) {
   return S_OK;
 }
 
+STDMETHODIMP ModelInfo::SetName(const char* name) {
+  name_ = std::string(name);
```

Comment:
Should the expectation of the user of the API be to also change the original protobuf?

Reply:
I think it should set it in the protobuf (in case the caller accesses it another way, the set name should change the in-memory model), but saving back to disk would be unexpected.

Reply:
Really? I think saving back to disk would be expected?